drive_cli/utils.py

```diff
@@ -369,10 +369,23 @@ def upload_file(name, path, pid):
         'parents': [pid],
         'mimeType': file_mimeType
     }
-    media = MediaFileUpload(path, mimetype=file_mimeType)
-    new_file = service.files().create(body=file_metadata,
-                                      media_body=media,
-                                      fields='id').execute()
+    if os.stat(path).st_size <= 256 * 1024:
```

Comment:
@h98aaron  Thanks for the PR, It looks great. A question though.

Shouldn't this be `1024*1024`? The above command returns fize_size in _bytes_ and the idea is to chunk file above 1MB, the condition should be True for file_size less <= 1MB.
Let me know in case I missed something out.

/cc @nurdtechie98

Reply:
You're right. 
My stupid mistake made me realize that it was 256MB..

Reply:
Please make the change(1) and update the PR.